apps/app-article/src/App.js

```diff
@@ -389,6 +389,12 @@ class App extends Component {
             articleTitle.style.lineHeight = "100%";
         }
 
+        if (document.getElementsByClassName('preamble').length > 0) {
```

Comment:
Just a note: When using React, it's not advisable to directly manipulate the DOM (there is a possibility when `setState` is called, changes made like this are overridden). It would be better to, somehow, resize the fonts by making a `state` change (in combination of using css).

Reply:
You are right @vapaj , I will consider rewriting the increaseFontSize functionality next release! I guess it is going to be changed anyhow and some of its funcionality be managed by the native side!